hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMBlockDeletingService.java

```diff
@@ -202,12 +206,17 @@ public EmptyTaskResult call() throws Exception {
               }
             }
           }
+          long totalTime = Time.monotonicNow() - startTime;
           LOG.info("Totally added {} blocks to be deleted for"
                   + " {} datanodes / {} totalnodes, task elapsed time: {}ms",
               transactions.getBlocksDeleted(),
               transactions.getDatanodeTransactionMap().size(),
               included.size(),
-              Time.monotonicNow() - startTime);
+              totalTime);
+          metrics.setStartTimeOfLastIteration(startTime);
+          metrics.setDurationOfLastIteration(totalTime);
+          metrics.setNumBlockDeletionCommandSentInLastIteration(commandsSent);
```

Comment:
its not useful to capture LastIteration metrics, as chart can be prepared with existing metrics itself. This metrics is not required.